src/Bridge/Doctrine/EventListener/PublishMercureUpdatesListener.php

```diff
@@ -49,11 +52,13 @@ final class PublishMercureUpdatesListener
     private $updatedObjects;
     private $deletedObjects;
     private $formats;
+    private $graphQlSubscriptionManager;
+    private $graphQlMercureSubscriptionUriGenerator;
 
     /**
      * @param array<string, string[]|string> $formats
      */
-    public function __construct(ResourceClassResolverInterface $resourceClassResolver, IriConverterInterface $iriConverter, ResourceMetadataFactoryInterface $resourceMetadataFactory, SerializerInterface $serializer, array $formats, MessageBusInterface $messageBus = null, callable $publisher = null, ExpressionLanguage $expressionLanguage = null)
+    public function __construct(ResourceClassResolverInterface $resourceClassResolver, IriConverterInterface $iriConverter, ResourceMetadataFactoryInterface $resourceMetadataFactory, SerializerInterface $serializer, array $formats, MessageBusInterface $messageBus = null, callable $publisher = null, ?GraphQlSubscriptionManagerInterface $graphQlSubscriptionManager = null, ?GraphQlMercureSubscriptionUriGeneratorInterface $graphQlMercureSubscriptionUriGenerator = null, ExpressionLanguage $expressionLanguage = null)
```

Comment:
The expression language is not defined in the configuration so it would mean doing dirty tricks.
Since it's experimental, I thought it would be better to not prevent the BC.